src/test/lambda/explorer/lambdaNodes.test.ts

```diff
@@ -5,119 +5,109 @@
 async function* asyncGenerator<T>(items: T[]): AsyncIterableIterator<T> {
     yield* items
 }
 
+const FAKE_REGION_CODE = 'someregioncode'
+const UNSORTED_TEXT = ['zebra', 'Antelope', 'aardvark', 'elephant']
+const SORTED_TEXT = ['aardvark', 'Antelope', 'elephant', 'zebra']
+
 describe('LambdaNode', () => {
-    class FunctionNamesMockLambdaClient extends MockLambdaClient {
-        public constructor(
-            public readonly functionNames: string[] = [],
-            listFunctions: () => AsyncIterableIterator<Lambda.FunctionConfiguration> = () =>
-                asyncGenerator<Lambda.FunctionConfiguration>(
-                    functionNames.map<Lambda.FunctionConfiguration>(name => {
-                        return {
-                            FunctionName: name
-                        }
-                    })
-                )
-        ) {
-            super({
-                listFunctions
-            })
-        }
-    }
+    let sandbox: sinon.SinonSandbox
+    let testNode: LambdaNode
 
-    class ThrowErrorLambdaNode extends LambdaNode {
-        public constructor() {
-            super('someregioncode')
-        }
+    // Mocked Lambda Client returns Lambda Functions for anything listed in lambdaFunctionNames
+    let lambdaFunctionNames: string[]
 
-        public async updateChildren(): Promise<void> {
-            throw new Error('Hello there!')
-        }
-    }
+    beforeEach(() => {
+        sandbox = sinon.createSandbox()
 
-    it('Sorts Lambda Function Nodes', async () => {
-        const inputFunctionNames: string[] = ['zebra', 'Antelope', 'aardvark', 'elephant']
+        lambdaFunctionNames = ['function1', 'function2']
 
-        // TODO: Move to MockToolkitClientBuilder
-        ext.toolkitClientBuilder = {
-            createCloudFormationClient(regionCode: string): CloudFormationClient {
-                throw new Error('cloudformation client unused')
-            },
+        initializeClientBuilders()
 
-            createEcsClient(regionCode: string): EcsClient {
-                throw new Error('ecs client unused')
-            },
+        testNode = new LambdaNode(FAKE_REGION_CODE)
+    })
 
-            createLambdaClient(regionCode: string): LambdaClient {
-                return new FunctionNamesMockLambdaClient(inputFunctionNames)
-            },
+    afterEach(() => {
+        sandbox.restore()
+    })
 
-            createStsClient(regionCode: string): StsClient {
-                throw new Error('sts client unused')
-            }
-        }
+    // TODO : Lambda Nodes do not show a placeholder when no Functions could be found
+    // it('returns placeholder node if no children are present', async () => {
+    //     lambdaFunctionNames = []
+
+    //     const childNodes = await testNode.getChildren()
 
-        const lambdaNode = new LambdaNode('someregioncode')
+    //     assertNodeListOnlyContainsPlaceholderNode(childNodes)
+    // })
```

Comment:
Offline conversation confirmed that this is a good test for functionality that doesn't exist. Please create an issue and add it to the comment so we remember